.github/workflows/ci.yml

```
@@ -115,8 +115,9 @@ jobs:
 
             - name: Frontend install-dependencies
               run: |
-                  vendor/bin/console frontend:mp:install-dependencies
-                  vendor/bin/console frontend:zed:install-dependencies
+                  vendor/bin/console frontend:project:install-dependencies
+                  vendor/bin/console frontend:yves:install-dependencies
```

Also, the `node-version` version must be changed from `12` to `16`.
```suggestion
                  vendor/bin/console frontend:project:install-dependencies -vvv
```